go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/ipfs/fs-repo-migrations
 
 go 1.13
+
+require golang.org/x/net v0.0.0-20200114155413-6afb5195e5aa
```

Comment:
Let's still vendor this, just share it. The migrations should build even if all external deps go down.

Reply:
ok, I vendored this, and I made sure .travis.yml is building with vendored deps. If that works, it means we should be able to build migrations offline if needed.